log.go

```diff
@@ -2,11 +2,45 @@ package wrapplog
 
 import (
 	"os"
+	"path"
 
 	"github.com/Sirupsen/logrus"
 )
 
+type WrappHook struct {
+	ServiceName string
+}
+
+func (w *WrappHook) Levels() []logrus.Level {
+	return logrus.AllLevels
+}
+
+func (w *WrappHook) Fire(entry *logrus.Entry) error {
```

Comment:
This function add a new field in log entries. The reason I did this is that some projects are using default log (e.g. tiles) and, since we require the field 'service', I was trying to find a way to add this field with a minimal code change. Current version of logrus doesn't support configuration of default fields but allows changing of log entries through hooks.